Make xdg_user_dir_lookup() non-static, so that Chromium can call it
through base/third_party/xdg_user_dirs/xdg_user_dir_lookup.h.

Chromium commit f877ff397ecc4 (https://codereview.chromium.org/27186).
Upstream has made the function non-static since d5559e0 (v0.20).

diff --git a/src/xdg_user_dir_lookup.cc b/src/xdg_user_dir_lookup.cc
--- a/src/xdg_user_dir_lookup.cc
+++ b/src/xdg_user_dir_lookup.cc
@@ -187,7 +187,7 @@ error2:
  * The return value is newly allocated and must be freed with
  * free().
  **/
-static char *
+char *
 xdg_user_dir_lookup (const char *type)
 {
   char *dir, *home_dir, *user_dir;
